diff -up aiot-tools-main/setup.py.omv~ aiot-tools-main/setup.py
--- aiot-tools-main/setup.py.omv~	2022-10-31 15:57:48.569126613 +0100
+++ aiot-tools-main/setup.py	2022-10-31 16:27:34.124646568 +0100
@@ -30,7 +30,7 @@ setuptools.setup(
         ]},
     install_requires=[
         'aiot-bootrom @ git+https://gitlab.com/mediatek/aiot/bsp/aiot-bootrom#aiot_bootrom',
-        'gpiod==1.4.0',
+        'gpiod>=1.4.0',
         'oyaml',
         'packaging',
         'pyftdi',
